src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,21 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        final String errorMessage = "Can't get min value from list: " + numbers;
+
+        if (numbers.isEmpty()) {
+            throw new RuntimeException(errorMessage);
+        }
+
+        return Arrays.stream(numbers.stream()
+                        .collect(Collectors.joining(","))
+                        .split(","))
```

Comment:
```suggestion
        return Arrays.stream(numbers.split(","))
```
Redundant converting

Reply:
my input example -> {"5,30,100", "0,22,7", ...}

so i must connect this strings in single one before splitting 

am i wrong?